pkg/deploy/identity-provider/deployment_keycloak.go

```diff
@@ -46,14 +47,16 @@ var (
 		cmp.Comparer(func(x, y appsv1.Deployment) bool {
 			return x.Annotations["che.self-signed-certificate.version"] == y.Annotations["che.self-signed-certificate.version"] &&
 				x.Annotations["che.openshift-api-crt.version"] == y.Annotations["che.openshift-api-crt.version"] &&
-				x.Annotations["che.keycloak-ssl-required-updated"] == y.Annotations["che.keycloak-ssl-required-updated"]
+				x.Annotations["che.keycloak-ssl-required-updated"] == y.Annotations["che.keycloak-ssl-required-updated"] &&
+				x.Annotations["che.ca-certificates.version"] == y.Annotations["che.ca-certificates.version"]
 		}),
 	}
 	keycloakAdditionalDeploymentMerge = func(specDeployment *appsv1.Deployment, clusterDeployment *appsv1.Deployment) *appsv1.Deployment {
 		clusterDeployment.Spec = specDeployment.Spec
 		clusterDeployment.Annotations["che.self-signed-certificate.version"] = specDeployment.Annotations["che.self-signed-certificate.version"]
 		clusterDeployment.Annotations["che.openshift-api-crt.version"] = specDeployment.Annotations["che.openshift-api-crt.version"]
 		clusterDeployment.Annotations["che.keycloak-ssl-required-updated"] = specDeployment.Annotations["che.keycloak-ssl-required-updated"]
+		clusterDeployment.Annotations["che.ca-certificates.version"] = specDeployment.Annotations["che.ca-certificates.version"]
```

Comment:
Where are these values set in the first place ? And when is it used ?

Reply:
Removed. Leftovers from the prev commit.